### PR TITLE
Add and fix the reset attributes in various stats/status commands

### PR DESCRIPTION
**Pull request by nichamon**

### CODE REVIEW

**Comment by nichamon** [comment]
@nick-enoent Please review

**Comment by nichamon** [comment]
I'll change the pull request to 'ready' after Jim has tried this. I'll update the branch with the updated man pages for ldmsd_controller and ldmsctl later.

**Comment by nichamon** [comment]
@tom95858 This is ready to be merged.

File: ldms/src/ldmsd/ldmsd_strgp.c [hunk 1]
```diff
@@ -681,6 +681,7 @@ int ldmsd_strgp_update_prdcr_set(ldmsd_strgp_t strgp, ldmsd_prdcr_set_t prd_set)
 		if (!ref)
 			break;
 		LIST_INSERT_HEAD(&prd_set->strgp_list, ref, entry);
+		__atomic_fetch_add(&strgp->prdset_cnt, 1, __ATOMIC_SEQ_CST);
```
Comment (tom95858) [inline ldms/src/ldmsd/ldmsd_strgp.c:684]: I don't see where you ever decrement this count. How does this work with set_delete?

File: ldms/src/ldmsd/ldmsd_prdcr.c [hunk 2]
```diff
@@ -143,6 +143,7 @@ void __prdcr_set_del(ldmsd_prdcr_set_t set)
 	ldmsd_strgp_ref_t strgp_ref = LIST_FIRST(&set->strgp_list);
 	while (strgp_ref) {
 		LIST_REMOVE(strgp_ref, entry);
+		__atomic_fetch_sub(&strgp_ref->strgp->prdset_cnt, 1, __ATOMIC_SEQ_CST);
```
Comment (nichamon) [inline ldms/src/ldmsd/ldmsd_prdcr.c:146]: @tom95858  I decrement the set count in `__prdset_set_del()`.

**Pull request merged**